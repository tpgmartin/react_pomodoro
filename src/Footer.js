import React from 'react'

export default class Footer extends React.Component {
  render() {
    return <footer>
            Author <a href="https://github.com/tpgmartin" target="_blank">
                    @tpgmartin
                   </a>
           </footer>
  }
}
