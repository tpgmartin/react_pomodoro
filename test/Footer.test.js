import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../src/Footer'

test('<Footer />', () => {

    const component = renderer.create(<Footer></Footer>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()

})
