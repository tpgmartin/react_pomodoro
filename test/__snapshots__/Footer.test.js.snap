// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<footer>
  Author
  <a
    href="https://github.com/tpgmartin"
    target="_blank"
  >
    @tpgmartin
  </a>
</footer>
`;
